.github/workflows/gradle.yml

```diff
@@ -122,6 +122,12 @@ jobs:
       with:
         name: output
         path: output
+    - name: Upload Raw Test Results
```

Comment:
Add this to help debug in the future when the test result details are not usable.